src/shared/components/elements/buttons/button1/style.scss

```diff
@@ -0,0 +1,230 @@
+  display: inline-block; 
+  text-transform: uppercase;
+  &:hover{
+  	background: $darkYellow;
+  	color: $blue;
+  }
+  &::before{
+    @extend .vCenter;
+    right: 15px;
+  	display: block;
+  	content: '';
+  	background-image: url('/images/sprites/sprite-elements.png');
+  	background-Position: -8px -0px;
+	width: 8px;
+	height: 13px;
+	background-size: 23px 21px;
+  }
+  .title{
+  	@extend .vCenter;
+  }
+}
+.e{
+  position: relative;
+  background: $blue;
+  width: 225px;
+  font-size: 12px;
+  color: $yellow;
+  font: $fontABold;
+  height: 30px;
+  margin-bottom: 20px;
+  padding-left: 25px;
+  @include border-radius(15px)
+  display: inline-block; 
+  text-transform: uppercase;
+  &:hover{
+  	background: $darkBlue;
+  	color: $yellow;
+  }
+  &::before{
+    @extend .vCenter;
+    right: 15px;
+  	display: block;
+  	content: '';
+  	background-image: url('/images/sprites/sprite-elements.png');
+  	background-Position: -16px -0px;
+	width: 7px;
+	height: 13px;
+	background-size: 23px 21px;
+  }
+  .title{
+  	@extend .vCenter;
+  }
+}
+.f{
+  position: relative;
+  background: $red;
+  width: 225px;
+  color: $yellow;
+  font: $fontABold;
+  font-size: 12px;
+  height: 30px;
+  margin-bottom: 20px;
+  padding-left: 25px;
+  @include border-radius(15px)
+  display: inline-block; 
+  text-transform: uppercase;
+  &:hover{
+  	background: $darkRed;
+  	color: $yellow;
+  }
+  &::before{
+    @extend .vCenter;
+    right: 15px;
+  	display: block;
+  	content: '';
+  	background-image: url('/images/sprites/sprite-elements.png');
+  	background-Position: -16px -0px;
+	width: 7px;
+	height: 13px;
+	background-size: 23px 21px;
+  }
+  .title{
+  	@extend .vCenter;
+  }
+}
+.g{
+  position: relative;
+  background: $yellow;
+  width: 225px;
+  color: $blue;
+  font: $fontABold;
+  font-size: 12px;
+  height: 30px;
+  margin-bottom: 20px;
+  padding-left: 25px;
+  @include border-radius(15px)
+  display: inline-block; 
+  text-transform: uppercase;
+  &:hover{
+    background: $darkYellow;
+    color: $blue;
+  }
+  &::before{
+    @extend .vCenter;
+    right: 15px;
+    display: block;
+    content: '';
+    background-image: url('/images/sprites/sprite-elements.png');
+    background-Position: -8px -0px;
+  width: 8px;
+  height: 13px;
+  background-size: 23px 21px;
+  }
+  .title{
+    @extend .vCenter;
+  }
+}
+.h{
+  position: relative;
+  background: $lightGray;
+  width: 185px;
+  color: $red;
+  font: $fontABold;
+  height: 55px;
+  margin-bottom: 20px;
+  padding-left: 65px;
+  @include border-radius(30px)
+  display: inline-block; 
+  &:hover{
+  	background: $white;
+  	color: $red;
+  }
+  &::before{
+    @extend .vCenter;
+  	display: block;
+  	content: '';
+  	background-image: url('/images/sprites/sprite-elements.png');
+  	background-Position: -16px -0px;
+	width: 7px;
+	height: 13px;
+	background-size: 23px 21px;
+  }
+  .title{
+  	@extend .vCenter;
+  }
+}
+@media screen and (max-width: 400px){
```

Comment:
en constans.scss hay una variable de $isPhone, $isTable ... ha que crear una para remplazar "screen and (max-width: 400px)", es mejor manejarlo a través de variable